index.d.ts

```diff
@@ -82,8 +82,9 @@ declare namespace websocketPlugin {
   }
 
   export interface PluginOptions {
-    handle: (connection: SocketStream) => void;
+    handle: (connection: SocketStream | WebSocket) => void;
```

Comment:
Just like the connection parameter here, the connection parameter on the `WebsocketHandler` type above should also be changed to be `SocketStream | WebSocket`, no?

Reply:
That's a good point I will investigate with some tests